docs/config.rst

```diff
@@ -214,6 +258,34 @@ The following configuration shows a bunch of mapping examples:
             </doctrine_mongodb:config>
         </container>
 
+    .. code-block:: php
+
+        use Symfony\Config\DoctrineMongodbConfig;
+
+        return static function (DoctrineMongodbConfig $config): void {
+            $config->documentManager('default')
+                ->mapping('MyBundle1')
+                ->mapping('MyBundle2')
+                    ->type('xml')
+                ->mapping('MyBundle3')
+                    ->type('attribute')
+                    ->dir(__DIR__.'/../src/Document')
+                ->mapping('MyBundle4')
+                    ->type('xml')
+                    ->dir(__DIR__.'/../config/doctrine/mapping')
+                ->mapping('MyBundle5')
+                    ->type('xml')
+                    ->dir('my-bundle-mappings-dir')
+                    ->alias('BundleAlias')
+                ->mapping('MyBundle6')
+                    ->type('xml')
+                    ->dir('%kernel.project_dir%/src/vendor/DoctrineExtensions/lib/DoctrineExtensions/Documents')
```

Comment:
I see that Symfony provides a `param` function that can be used instead `%notation%`. Shall we endorse the new way or is there no community consensus yet?

Reply:
Yes, I'll use it.